src/transaction/types.ts

```diff
@@ -9,18 +9,30 @@ export const enum TxWitnessKeys {
   REDEEMERS = 5,
 }
 
-export type TxWitnessByron = [
+export type TxWitnessByronData = [
   Buffer,
   Buffer,
   Buffer,
   Buffer,
```

Comment:
Don't these deserve a name? (Also below for Shelley.)

Reply:
We have these structures so that we can directly pass them to the CBOR serialization lib and write the result to a file, so I don't find storing the data as an object and restructuring it a few function calls later that useful (since we never access the data later on). What about adding comments to the items so that it's clear what they represent (https://github.com/vacuumlabs/cardano-hw-cli/pull/122/commits/6d172f24495960a7f0ea1e638ad3ebe62123f336)?

Reply:
The comments are good enough.